coding-concepts.js

```diff
@@ -16,44 +16,44 @@ const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2022"
-// console.log(cohort.length)
+//console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 10
+// b) Verify and explain: The answer is 10 becasue .length is looking for the number of characters which includes the space
 
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
-// console.log(greeting[4])
+//console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: o
+// b) Verify and explain: the answer is o because it is looking for the index of 4, so you count the characters starting from 0
 
 
 // --------------------3) What will this log?
 
 const languages = ["JavaScript", "Ruby", "Python", "C++"]
-const index = 1
-// console.log(languages[index])
+//const index = 2
+//console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Ruby
+// b) Verify and explain: the answer is Ruby because it is asking for the index of 1, so when you count each word (you count the words not letters because each word is in "") starting from 0, Ruby is in position 1
 
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
+console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: SATURDAY, SUNDAY
+// b) Verify and explain: .toUpperCase can only be used on strings. In order for you to use it with an array, you would need to index which string you wanted to capitalize 
```

Comment:
Awesome!  This one is certainly tricky.  You could also use iteration such as a for loop to modify all values in the array.